pyproject.toml

```diff
@@ -1,16 +1,15 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "lsst-meas-extensions-multiprofit"
+name = "lsst-multiprofit"
```

Comment:
@taranu I think this is a cut and paste error. The name was right initially and with my recent changes to sconsUtils this name now clashes with multiprofit itself. I think the description is also incorrect since this is a plugin for meas_base and not the source model fitting itself. I'll file a ticket and fix the name at least.

Reply:
Whoops. Well yes it is an error; I guess nothing else actually uses the project name for anything?

However, this is not actually a measurement framework plugin. Those are all single band. The tasks in this package are separate multiband tasks. The name is following the convention set by meas_extensions_scarlet (there was some discussion of dropping the _extensions part but that didn't happen, for better or worse).

Reply:
The problem is that the name is used now. I will fix it. The "requires" section looks wrong to me as well.